service/src/main/kotlin/fi/hameenkyro/evaka/HameenkyroConfig.kt

```diff
@@ -48,7 +50,34 @@ class HameenkyroConfig {
         preferredStartRelativeApplicationDueDate = true,
         fiveYearsOldDaycareEnabled = false,
         archiveMetadataOrganization = "Hämeenkyrön kunnan varhaiskasvatus",
```

Comment:
Also update this `archiveMetadataOrganization` for every municipality.

Reply:
Organization names updated.